opennlp-tools/src/main/java/opennlp/tools/namefind/TokenNameFinderModel.java

```diff
@@ -209,6 +209,7 @@ protected void createArtifactSerializers(Map<String, ArtifactSerializer> seriali
     Map<String, ArtifactSerializer> serializers = BaseModel.createArtifactSerializers();
 
     serializers.put("featuregen", new ByteArraySerializer());
+    serializers.put("bin", new ByteArraySerializer());
```

Comment:
This should all be removed (maybe tricky with old models) but at least not used anymore.

The way to do that today is GeneratorFactory.extractCustomArtifactSerializerMappings the method has to be changed a bit to take all feature generators which have resources into account. Also the code in TokenNameFinderTrainerTool should then work.